### PR TITLE
PhpdocAlignFixer - don't crash poorly formatted phpdoc

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I've implemented a quick fix that should stop it crashing. A better fix would be to have it still able to fix such lines, but this'll do for now.

**Comment by GrahamCampbell** [comment]
@keradus What do you make of this please?

File: src/Fixer/Phpdoc/PhpdocAlignFixer.php [hunk 1]
```diff
@@ -113,7 +113,17 @@ private function fixDocBlock($content)
             $current = $i;
             $items[] = $matches;
 
-            while ($matches = $this->getMatches($lines[++$i], true)) {
+            while (true) {
+                if (!isset($lines[++$i])) {
+                    break 2;
```
Comment (keradus) [inline src/Fixer/Phpdoc/PhpdocAlignFixer.php]: :(

**Comment by keradus** [comment]
please rebase on top of 2.0 branch

**Comment by GrahamCampbell** [comment]
Reset to 2.0, and cherry-picked my commits.

**Comment by keradus** [comment]
Thank you @GrahamCampbell.

**Pull request closed without merging**